### PR TITLE
feature: add new docs about navigation and refresh mechanism 

### PR DESCRIPTION
**Pull request by snooky23**
## Description
Add new docs about navigation and refresh mechanism 

## Checklist

* [X] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation is included

* [ ] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+2. Reload the current data model and refresh the screen. The `shouldReloadStack` boolean variable determines whether to refresh all `pushed` app stack view controllers or just the current top view controller.
+
+    ```swift
+    ZAAppConnector.sharedInstance().navigationDelegate?.reloadCurrentScreen(shouldReloadStack: Bool)
+    ```
+
+## Android Implementation
+
+To use the following utility methods you first need to do the following:
```
Comment (AlexDmitryukov) [inline book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md]: Following used twice in one sentence.

File: book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+1. In your plugin gradle file add a dependency to `applicaster-android-sdk`. Please note that the minimum version required is v4.2.6.
```
Comment (AlexDmitryukov) [inline book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md]: may be `v4.2.6` should be **bold** to catch attention?

File: book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+1. Clean the app screens stack and open a screen according to `screenId` parameter. The behavior is equivalent to restart app without showing a splash screen and open the relevant screen.
+
+    ```swift
+    AppRedirectReceiver.INSTANCE.redirect(screenId)
+    ```
+
+2. Clean the app screens stack and open `Home` screen. The behavior is equivalent to restart app without showing a splash screen and open the Home screen.
+
+    ```swift
+    AppRedirectReceiver.INSTANCE.fun redirectHome()
```
Comment (AlexDmitryukov) [inline book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md]: `AppRedirectReceiver.INSTANCE.redirectHome()`

File: book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+### Redirect
+
+The `AppRedirectReceiver` class provides a redirect functionality for GenericBaseFragmentActivity
+
+See below the main useable methods:
+
+1. Clean the app screens stack and open a screen according to `screenId` parameter. The behavior is equivalent to restart app without showing a splash screen and open the relevant screen.
+
+    ```swift
```
Comment (AlexDmitryukov) [inline book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md]: please remove this foul language from android section :)

File: book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md (hunk 1 above)
Comment (snooky23) [reply book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md]: Done

File: book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md (hunk 2 above)
Comment (snooky23) [reply book/plugins/general-abilities/refresh-and-navigation/refresh-and-navigation.md]: done

**Pull request merged**